.storybook/custom/components/Import.js

```diff
@@ -11,7 +11,7 @@ const Import = () => {
         customStyle={{ padding: 10, whiteSpace: 'pre-wrap', fontSize: 14 }}
         language='javascript'
         style={googlecode}>
-        {`import { ${groups[groups.length - 1]} } from 'fundamental-react/${groups[1]}';`}
```

Comment:
So this isn't your fault, but I just noticed:

`import { ButtonSegmented } from 'fundamental-react/lib/ButtonSegmented';` is not correct, it should be `import { ButtonSegmented } from 'fundamental-react/lib/Button';`

I don't think there's any other components that share a folder

This works for forms because they're inside of a folder in storybook. So we can either make this import component smarter or fix Button to be under a folder by changing https://github.com/SAP/fundamental-react/blob/master/src/Button/__stories__/ButtonSegmented.stories.js#L7 to `title: 'Component API/Button/ButtonSegmented'` and the same for Button.

Reply:
```
import { TabGroup } from 'fundamental-react/lib/TabGroup';
```

should be
```
import { TabGroup } from 'fundamental-react/lib/Tabs';
```